src/document-submit.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getService} from './service';
+
+/**
+ * @param {!Window} window
+ */
+export function installGlobalSubmitListener(window) {
+  submitHandlerFor(window);
+}
+
+/**
+ * @param {!Window} window
+ */
+export function uninstallGlobalSubmitListener(window) {
```

Comment:
Do we just store a local variable in this file and memoize it? How do we handle multiple window situations? Would something like the following be ok?

```Javascript
/**
 * @type {?SubmitHandler} Store the global handler for form submits.
 */
let submitHandler_;

/**
 * @param {!Window} window
 */
export function installGlobalSubmitListener(window) {
  submitHandlerFor(window);
}

/**
 * @param {!Window} window
 */
function submitHandlerFor(window) {
  return submitHandler_ || (submitHandler_ = new SubmitHandler(window));
}
```